### DISCUSSION TITLE
How to change the chord symbol's instrument in Musescore 4?

### DISCUSSION THREAD

**Discussion by joon060707** [musescore/MuseScore#20186] [Q&A]
(Sorry for my English as a foreign speaker)

Below is the image from Musescore 3.
![MS3](https://github.com/musescore/MuseScore/assets/71119608/ac6f212d-d7fc-478e-9836-deffbcacb180)

In Musescore 3, I could change the chord instrument;
For example, assume that the default instrument is Grand Piano.
I entered several notes (i.e., 𝅘𝅥𝅮) and chord symbols (i.e., C, Bm....)
If I played the sheet, both notes and chord would be played by Grand Piano.
However, I could change the chord symbol's instrument by expanding the mixer and through the "Sound" tab.
Now, the chord symbol on the Grand Piano would be played by "Flute".

However, 
![MS4](https://github.com/musescore/MuseScore/assets/71119608/04d6d652-c07f-4c22-bd66-69a0e47eac71)
in Musescore 4, there was no musical instrument selection in the "Sound" tab.
It shows only soundfonts(MS Basic).

When I changed the chord instrument in the Musescore 3, it works.
But, if I opened that Musescore 3 file on Musescore 4, it was played using its default instrument.
(The chord symbol could be played through "Flute" on Musescore 3 but through "Grand Piano" on Musescore 4.)
So, I could assume that that function might be gone on Musescore 4.

Did anyone know how to change the chord symbol's imstrument in Musescore 4?
If available, please share your method.
If not, I might use Musescore 3 instead, or please add this function.

Thank you for your team's effort.

**Comment by iwoithe** [comment]
Hey @joon060707 👋

I'm pretty sure the issue you've described here should be resolved by https://github.com/musescore/MuseScore/pull/18976 🙂

**Reply by joon060707** [reply]
Additional info: the version is 4.1.1.232071203

**Reply by joon060707** [reply]
@iwoithe
I already read such pages (#18976, #17233, #15190) before writing it, so could I understand that the issue might be resolved in the version 4.2??

Thanks.

**Reply by iwoithe** [reply]
Apologies for forgetting to mention that, yes I believe that it will be in the 4.2 release 🙂

**Comment by MarcSabatella** [comment]
Meanwhile, you can check out the beta of 4.2 [here](https://musescore.org/en/node/357135).  And, you can also select individual sounds for chord symbols - or any instrument - from the free Msue Sounds, which your screen shot shows you haven't installed yet.  Simply install those and you can have flutes play your chords if you like.  You can also install additional free soundfonts for other sounds, such as [my "split" version of MS Basic](https://community.masteringmusescore.com/c/resources/ms-basic-individual-sounds) that allows individual access to all of its sounds already.  So, no need to wait for 4.2.

For more help with using MuseScore, please ask on the official Support forum over on musescore.org
